### ISSUE TITLE
Support ES 0.90.0 version

### ISSUE THREAD

**Issue by piterimov** [bleskes/elasticfacets#7]
Hello,

It seems that the plugin doesn't work with ElasticSearch 0.90.0 version at the moment.
It would be nice to have this fixed if possible.

Thanks!

**Comment by bleskes** [comment]
Hi,

Yes, sadly some of the plugin features in master (mostly the hashed string
facet implementation) can not be ported to the 0.90.X branch because the
internal changes have made it impossible. I'm still debating whether I
should just throw them away or wait until they can be done. Is this urgent
for you? I might try to find time to clear it out and have a trimmed down
version which support 0.90.x . No promises though :)

Boaz

Boaz

**Comment by piterimov** [comment]
Hello, thanks for your reply!
At the moment I'm interested in faceted date histogram feature.
This is not too urgent for me at the moment.
But I would really appreciate any opportunity to use this with 0.9 version.

Thanks!

**Comment by diranl** [comment]
+1, it would be great if you can port it to 0.90, this plugin does exactly what I need!

**Comment by chrishartfield** [comment]
+1, the faceted date histogram is really a great tool and at least that part should be allowed on 0.90+

**Comment by fredbenenson** [comment]
I'd love to have the faceted date histogram supported in 0.90. It looks like elasticsearch 1.0 is going to be be rethinking facets into aggregations that should help: https://github.com/elasticsearch/elasticsearch/issues/3300

**Comment by mattweber** [comment]
@fredbenenson this will not be needed with aggregations as you can nest multiple aggregations.  For example, you would use the date histogram aggregator (ex. by week) and then nest an terms aggregator and you will have the top terms broken down by week which is exactly what the faceted date histogram does.

**Comment by fredbenenson** [comment]
@mattweber yes, sorry, I was unclear -- that was exactly my point. "should help" == "should be obsoleted by"

**Comment by n0rthwood** [comment]
Would actually be very interested to see a working branch for the date histogram facet to be working against 0.9
